packages/authentication/src/definitions.ts

```diff
@@ -326,6 +331,11 @@ export interface User {
   uid: string;
 }
 
+export interface AdditionalUserInfo {
+  userId?: string | null;
+  email?: string | null;
```

Comment:
```suggestion
  isNewUser: boolean;
  username?: string;
  profile?: { [key: string]: any; };
```

It's best to follow the official JS SDK for this interface: https://firebase.google.com/docs/reference/js/auth.additionaluserinfo#properties